packages/controllers/profile_edit.coffee

```diff
@@ -0,0 +1,32 @@
+if Meteor.isClient
+  Template.profileEdit.onCreated ->
+    @subscribe('userProfile')
+
+  Template.profileEdit.helpers
+    userProfile: ->
+      UserProfiles.findOne()
+
+  Template.profileEdit.events
+    'submit form': (event) ->
+      event.preventDefault()
+      form = event.target
+      fields = {
+        fullName: form.fullName?.value
+        jobTitle: form.jobTitle?.value
+        bio: form.bio?.value
+        emailHidden: form.emailHidden?.checked
+      }
+      Meteor.call 'updateProfile', fields, (error, response) ->
+        if error
+          toaster.error("Error")
```

Comment:
I should have removed this line anyway since there's no test for it yet (and no way to actually cause this error, which is why I missed the typo).